test/system/src/main/java/io/pravega/test/system/framework/services/PravegaSegmentStoreService.java

```diff
@@ -127,20 +127,22 @@ private App createPravegaSegmentStoreApp() {
         //set env
         String zk = zkUri.getHost() + ":" + ZKSERVICE_ZKPORT;
 
-        //System properties to configure SS service.
-        String hostSystemProperties = setSystemProperty("pravegaservice.zkURL", zk) +
-                setSystemProperty("bookkeeper.zkAddress", zk) +
-                setSystemProperty("hdfs.hdfsUrl", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020") +
-                setSystemProperty("autoScale.muteInSeconds", "120") +
+        //Environment variables to configure SS service.
+        Map<String, String> map = new HashMap<>();
+        map.put("ZK_URL", zk);
+        map.put("BK_ZK_URL", zk);
+        map.put("HDFS_URL", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020");
+        map.put("CONTROLLER_URL", conUri.toString());
+        map.put("TIER2_STORAGE", "HDFS");
+
+        //Properties set to override defaults for system tests
+        String hostSystemProperties = setSystemProperty("autoScale.muteInSeconds", "120") +
```

Comment:
In a normal production run these environment variables are set. `entrypoint.sh` picks these up and interpretes these. System tests were taking a different path and setting `PRAVEGA_SEGMENTSTORE_OPTS` env variable. This is unnecessary divergence from production runs. With these changes, we set the variables that are expected by `entrypoint.sh`. System test specific configs are set using `PRAVEGA_SEGMENTSTORE_OPTS`.